src/lib/bot/mod.rs

```diff
@@ -212,21 +254,24 @@ async fn playlist_queue(bot: &mut Bot, evt: &messages::Privmsg<'_>, args: Option
                 for e in errors {
                     error!("=> Error: {}", e);
                 }
-                return (format!("THATSREALLYTOODANK failed to queue the playlist"), true);
+                return (format!("cheemSad Failed to queue the playlist"), true);
             }
         },
         Err(e) => {
             error!("Failed to retrieve the videos in the playlist: {}", e);
-            return (format!("WAYTOODANK devs broke something"), true);
+            return (format!("cheemSad Failed to queue the playlist"), true);
         }
     }
 }
 
-type ResponseFactory = for<'a> fn(&'a mut Bot, evt: &'a messages::Privmsg<'_>, Option<Vec<&'a str>>) -> Pin<Box<dyn Future<Output = (String, bool)> + 'a>>;
+type ResponseFactory = for<'a> fn(
+    &'a mut Bot,
+    evt: &'a messages::Privmsg<'_>,
+    Option<Vec<&'a str>>,
+) -> Pin<Box<dyn Future<Output = (String, bool)> + 'a>>;
```

Comment:
Consider using a `Result` instead? I think that we should make all commands return a `Result<Message, BoxedError>` so we can a) Use `?` for error handling and b) have all errors stop propagating at the same time and in the same place.